```
/home/runner/work/mod-morph-all-players/mod-morph-all-players/modules/mod-morph-all-players/src/morph_all_command.cpp:34:9: fatal error: unknown type name 'SessionMap'
   34 |         SessionMap const& m_sessions = sWorld->GetAllSessions();
      |         ^
1 error generated.
```

--- a/src/morph_all_command.cpp
+++ b/src/morph_all_command.cpp
@@ -34,7 +34,7 @@ public:
 
         WorldSessionMgr::SessionMap const& m_sessions = sWorldSessionMgr->GetAllSessions();
 
-        for (SessionMap::const_iterator itr = m_sessions.begin(); itr != m_sessions.end(); ++itr)
+        for (WorldSessionMgr::SessionMap::const_iterator itr = m_sessions.begin(); itr != m_sessions.end(); ++itr)
         {
             if (!itr->second || !itr->second->GetPlayer() || !itr->second->GetPlayer()->IsInWorld())
                 continue;
@@ -53,7 +53,7 @@ public:
     {
         WorldSessionMgr::SessionMap const& m_sessions = sWorldSessionMgr->GetAllSessions();
 
-        for (SessionMap::const_iterator itr = m_sessions.begin(); itr != m_sessions.end(); ++itr)
+        for (WorldSessionMgr::SessionMap::const_iterator itr = m_sessions.begin(); itr != m_sessions.end(); ++itr)
         {
             if (!itr->second || !itr->second->GetPlayer() || !itr->second->GetPlayer()->IsInWorld())
                 continue;

--- a/src/morph_all_command.cpp
+++ b/src/morph_all_command.cpp
@@ -32,7 +32,7 @@ public:
         if (!displayId)
             return false;
 
-        SessionMap const& m_sessions = sWorldSessionMgr->GetAllSessions();
+        WorldSessionMgr::SessionMap const& m_sessions = sWorldSessionMgr->GetAllSessions();
 
         for (SessionMap::const_iterator itr = m_sessions.begin(); itr != m_sessions.end(); ++itr)
         {
@@ -51,7 +51,7 @@ public:
 
     static bool HandleDeMorphAllCommand(ChatHandler* /*handler*/)
     {
-        SessionMap const& m_sessions = sWorldSessionMgr->GetAllSessions();
+        WorldSessionMgr::SessionMap const& m_sessions = sWorldSessionMgr->GetAllSessions();
 
         for (SessionMap::const_iterator itr = m_sessions.begin(); itr != m_sessions.end(); ++itr)
         {

--- a/src/morph_all_command.cpp
+++ b/src/morph_all_command.cpp
@@ -5,6 +5,7 @@
 #include "Player.h"
 #include "ScriptMgr.h"
 #include "World.h"
+#include "WorldSessionMgr.h"
 
 using namespace Acore::ChatCommands;
 
@@ -31,7 +32,7 @@ public:
         if (!displayId)
             return false;
 
-        SessionMap const& m_sessions = sWorld->GetAllSessions();
+        SessionMap const& m_sessions = sWorldSessionMgr->GetAllSessions();
 
         for (SessionMap::const_iterator itr = m_sessions.begin(); itr != m_sessions.end(); ++itr)
         {
@@ -50,7 +51,7 @@ public:
 
     static bool HandleDeMorphAllCommand(ChatHandler* /*handler*/)
     {
-        SessionMap const& m_sessions = sWorld->GetAllSessions();
+        SessionMap const& m_sessions = sWorldSessionMgr->GetAllSessions();
 
         for (SessionMap::const_iterator itr = m_sessions.begin(); itr != m_sessions.end(); ++itr)
         {

